lib/game_of_life_web/views/universe_view.ex

```diff
@@ -21,6 +22,10 @@ defmodule GameOfLifeWeb.UniverseView do
     end
   end
 
+  def render_color_rgb(%Color{} = color) do
```

Comment:
you could expand pattern matching here:

```elixir
def render_color_rgb(%Color{red: red, green: green, blue: blue}), do: "rgb(#{red},#{green},#{blue})"
```

Reply:
you could also add a function to the `Color` struct that generates this string

Reply:
Oooh I like that!

Reply:
Oh right, interpolation is a thing...